bagitobjecttransfer/recordtransfer/views.py

```diff
@@ -322,54 +323,124 @@ class TransferFormWizard(SessionWizardView):
         },
     }
 
-    def get(self, request, *args, **kwargs):
-        resume_id = request.GET.get("resume_transfer", None)
+    def dispatch(self, request, *args, **kwargs):
+        self.in_progress_uuid = request.GET.get("transfer_uuid")
+        self.submission_group_uuid = None
+        if not self.in_progress_uuid:
+            self.submission_group_uuid = request.GET.get("group_uuid")
+        return super().dispatch(request, *args, **kwargs)
 
-        if resume_id:
-            transfer = SavedTransfer.objects.filter(user=self.request.user, id=resume_id).first()
+    def get(self, request, *args, **kwargs):
+        if self.in_progress_uuid:
+            transfer = get_in_progress_submission(request.user, self.in_progress_uuid)
```

Comment:
```suggestion
            transfer = InProgressSubmission.objects.filter(user=request.user, uuid=self.in_progress_uuid)
```

Reply:
Resolved by:
[Refactored view to use InProgressSubmssion instance variable rather t…](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/240/commits/ce4b0fa60c2c9ffb80c8229683ae4799309f983b)